readme/SOURCE_PLUGIN_README_TEMPLATE.md

```diff
@@ -10,9 +10,9 @@
 
 Here are the metrics collected by the plugin source.
 
-|Name|Type|Unit|Description|Attributes|More information|
-|----|----|----|-----------|----------|-----------------|
-|`metric's name`||||||
+|Name|Type|Unit|Description|Ressource|RessourceConsumer|Attributes|More information|
```

Comment:
Ressource => Resource
RessourceConsumer => ResourceConsumer

same for transform